packages/web/src/components/ResultTable.module.css

```
@@ -42,3 +42,16 @@
   padding: 24px 16px;
 }
+
+.transcription {
+  padding: 10px 16px;
+  font-size: 0.85rem;
+  border-top: 1px solid var(--color-border);
+}
+
+.transcription pre {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)